Makefile

```diff
@@ -1,5 +1,5 @@
 CFLAGS=
-OFLAGS=-pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -fPIC -I/usr/include/python2.7
+OFLAGS=-pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -fPIC -I/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
```

Comment:
This is mac specific, but we're not going to be running Make anywhere anyhow as the code currently sits.